core/primitives/extrinsic.hpp

```diff
@@ -15,7 +15,14 @@ namespace kagome::primitives {
   struct Extrinsic {
     kagome::common::Buffer data;  ///< extrinsic content as byte array
   };
-
 }  // namespace kagome::primitives
 
+namespace kagome::scale {
```

Comment:
same

Reply:
fixed for all written polkadot primitives operators <<